pkgs/development/compilers/adoptopenjdk-bin/jdk-darwin-base.nix

```diff
@@ -28,9 +28,6 @@ let cpuName = stdenv.hostPlatform.parsed.cpu.name;
     # Remove some broken manpages.
     rm -rf $out/Home/man/ja*
 
-    # for backward compatibility
-    ln -s $out/Contents/Home $out/jre
-
```

Comment:
Why is this not needed anymore?

Reply:
See https://github.com/NixOS/nixpkgs/pull/65247/files#r311605410